wake/actions/cart/updateItemQuantity.ts

```diff
@@ -48,7 +48,7 @@ const removeFromCart = (
     RemoveItemFromCartMutationVariables
   >({
     variables: {
-      input: { id: cartId, products: [{ ...props, quantity: 1e6 }] },
+      input: { id: cartId, products: [{ ...props }] },
```

Comment:
```suggestion
      input: { id: cartId, products: [props] },
```

Reply:
why did you remove quantity?

Reply:
the quantity was used to remove all products imagining that there would be less than 1000000, which is not necessary considering that I pass the exact value that should be removed